users/migrations/0003_auto_20190927_1338.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 2.2.5 on 2019-09-27 13:38
+
+from django.db import migrations
+import users.managers
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('users', '0002_user_organization'),
+    ]
+
+    operations = [
+        migrations.AlterModelManagers(
+            name='user',
+            managers=[
+                ('objects', users.managers.PhishtrayUserManager()),
+            ],
+        ),
+    ]
```

Comment:
how did this migration get here? :)
I can't find the matching code for this

Reply:
hehehe. When I change the manager for users, django created this migration... objects = PhishtrayUserManager()

Reply:
Removed.